### ISSUE TITLE
Federation pixelfed.social unable to see post and account

### ISSUE THREAD

**Issue by enfief** [pixelfed/pixelfed#743]
Hello, 
I am registered on pixelfed.social. When I am looking for my profil / posts from pixelfed on mastodon 
i can't find them and can't find my profil. 
But I can find some others users from the same instance pixelfed.social on mastodon. 

So I don't understand why we can see some members on the instance Z on mastodon but not all the members from the Z instance ?

**Comment by dansup** [comment]
@enfief This is how ActivityPub works. A mastodon/remote instance will only know about posts after a user from that instance follows a pixelfed account.

**Comment by enfief** [comment]
@dansup Thanks for your answer Dan, but I still can not understand. 
I am registered on your instance on pixelfed. When I am on mastodon I follow your profil and the profil of an another members of pixelfed.social instance but I still can not see my own post or profil of pixelfed. 
Why can I see yours and not mine? if we are on the same instance?

**Comment by trwnh** [comment]
@enfief Try copying the URL of the post and pasting it into the search bar. The post should load then, because you would be requesting it directly. For the reason why old posts don't appear in Mastodon, see also https://github.com/tootsuite/mastodon/issues/34 -- the issue is not with pixelfed.

**Comment by peacememories** [comment]
Sorry to post in an issue that's been closed for so long, but I still experience a version of this. I set up my own Pixelfed instance and followed my own Pixelfed account from a mastodon server. After I've directly looked at a post - using the URL method you described above - they are visible on mastodon, but before that, even new posts do not show up. The post count on the profile updates, though.
Maybe I've configured something incorrectly, but I cannot find any discrepancies.
